src/env.h

```diff
@@ -1489,6 +1489,14 @@ class Environment : public MemoryRetainer {
   template <typename T>
   void ForEachBaseObject(T&& iterator);
 
+  inline int64_t get_heap_snapshot_near_heap_limit() {
```

Comment:
I believe the naming convention for getter in cpp is that `field_name()` for getter and `set_field_name()` for setter.

```suggestion
  inline int64_t heap_snapshot_near_heap_limit() {
```

Reply:
Fixed.